packages/core/src/codewhisperer/activation.ts

```diff
@@ -271,6 +272,10 @@ export async function activate(context: ExtContext): Promise<void> {
     // we need to do it manually here because the Toolkit would have been unable to call
     // this API if the Q/CW extension started afterwards (and this code block is running).
     if (isExtensionInstalled(VSCODE_EXTENSION_ID.awstoolkit)) {
+        await activateExtension(VSCODE_EXTENSION_ID.awstoolkit)
```

Comment:
deadlock with https://github.com/aws/aws-toolkit-vscode/blob/feature/standalone/packages/core/src/amazonq/explorer/amazonQTreeNode.ts#L147

?

Reply:
This is removed in latest commit.